cmd/security_group_rule_add.go

```diff
@@ -31,14 +31,15 @@ type securityGroupAddRuleCmd struct {
 
 	SecurityGroup string `cli-arg:"#" cli-usage:"SECURITY-GROUP-ID|NAME"`
 
-	Description         string `cli-usage:"rule description"`
-	FlowDirection       string `cli-flag:"flow" cli-usage:"rule network flow direction (ingress|egress)"`
-	ICMPCode            int64  `cli-usage:"rule ICMP code"`
-	ICMPType            int64  `cli-usage:"rule ICMP type"`
-	Port                string `cli-usage:"rule network port (format: PORT|START-END)"`
-	Protocol            string `cli-usage:"rule network protocol"`
-	TargetNetwork       string `cli-flag:"network" cli-usage:"rule target network address (in CIDR format)"`
-	TargetSecurityGroup string `cli-flag:"security-group" cli-usage:"rule target Security Group NAME|ID"`
+	Description               string `cli-usage:"rule description"`
+	FlowDirection             string `cli-flag:"flow" cli-usage:"rule network flow direction (ingress|egress)"`
+	ICMPCode                  int64  `cli-usage:"rule ICMP code"`
+	ICMPType                  int64  `cli-usage:"rule ICMP type"`
+	Port                      string `cli-usage:"rule network port (format: PORT|START-END)"`
+	Protocol                  string `cli-usage:"rule network protocol"`
+	TargetNetwork             string `cli-flag:"network" cli-usage:"rule target network address (in CIDR format)"`
+	TargetSecurityGroup       string `cli-flag:"security-group" cli-usage:"rule target Security Group NAME|ID"`
+	TargetPublicSecurityGroup string `cli-flag:"public-security-group" cli-usage:"rule target Security Group NAME"`
```

Comment:
`cli-usage`:"rule target **Public** Security Group NAME" (to differ from `TargetSecurityGroup`).